tests/unit/package_managers/bundler/test_parser.py

```diff
@@ -0,0 +1,215 @@
+import subprocess
```

Comment:
Unit tests should be introduced along with the code that needs to be tested unlike integration tests which can come at the end because it tests functionality as a completely integrated unit.